jwst/cube_build/cube_build_wcs_util.py

```diff
@@ -62,7 +61,7 @@ def find_corners_MIRI(input, this_channel, instrument_info, coord_system):
         valid = np.where(yedge < 1023)
         xedge = xedge[valid]
         yedge = yedge[valid]
-        coord2b, coord1b, lamb = detector2alpha_beta(xedge,yedge) #lam ~0 for this transform
+        coord2b, coord1b, lamb = detector2alpha_beta(xedge,yedge)  # lam ~0 for this transform
```

Comment:
Huh, how did that sneak past the checker?

Reply:
Yeah, because we are not enforcing E231, space after the comma, I didn't run `autopep8` with this code.  Clearly in this case it would be preferable.  But this would also enforce slicing to be done with spaces after both `,` and `:`.

Reply:
Does E231 only apply to argument lists or anything with a comma?  Personally I think it's useful to enforce this one (makes the argument list easier for humans to parse).

Reply:
Unfortunately, E231 is an all-or-nothing check.  You either enforce spacing for arg lists and numpy array slicing or you don't.  It enforces a space after `,`, `:`, `;` anytime they're used.